proto/stream_plan.proto

```diff
@@ -504,7 +504,24 @@ message StreamScanNode {
   uint32 snapshot_read_barrier_interval = 9 [deprecated = true];
 
   // The external table that will be backfilled for CDC.
-  plan_common.ExternalTableDesc cdc_table_desc = 10;
+  plan_common.ExternalTableDesc cdc_table_desc = 10 [deprecated = true];
+}
+
+message StreamCdcScanNode {
+  uint32 table_id = 1;
+
+  // The columns from the upstream table that'll be internally required by this stream scan node.
+  // Contains Primary Keys and Output columns.
+  repeated int32 upstream_column_ids = 2;
+
+  // Strips the primary key columns if they're unnecessary.
+  repeated uint32 output_indices = 3;
+
+  /// The state table used by Backfill operator for persisting internal state
+  catalog.Table state_table = 4;
+
```

Comment:
I think we may also add a `rate_limit` field for `CdcScan`, in theory it can reuse the flow control executor.

Reply:
I will leave it out until we actually add support for it, since removing it later is a breaking change.